Dockerfile.web

```diff
@@ -0,0 +1,27 @@
+FROM node:10.16.3 as builder
```

Comment:
How locked are we to this version?  We could use the labels for the 'lts' releases https://hub.docker.com/_/node/

Reply:
I wanted to keep it the same as the other apps. I like the suggestion, How do you feel about me switching them all in a chore story?

Reply:
Is this good? https://www.pivotaltracker.com/story/show/171056321

Reply:
@felder-cl looks good. I'll add some step by step acceptance criteria